nym-api/src/network/handlers.rs

```diff
@@ -2,33 +2,59 @@
 // SPDX-License-Identifier: GPL-3.0-only
 
 use crate::network::models::{ContractInformation, NetworkDetails};
-use crate::{support::http::static_routes, v2::AxumAppState};
+use crate::v2::AxumAppState;
 use axum::{extract, Router};
 use nym_contracts_common::ContractBuildInformation;
 use std::collections::HashMap;
+use utoipa::ToSchema;
 
 pub(crate) fn nym_network_routes() -> Router<AxumAppState> {
     Router::new()
+        .route("/details", axum::routing::get(network_details))
+        .route("/nym-contracts", axum::routing::get(nym_contracts))
         .route(
-            &static_routes::v1::network::details(),
-            axum::routing::get(network_details),
-        )
-        .route(
-            &static_routes::v1::network::nym_contracts(),
-            axum::routing::get(nym_contracts),
-        )
-        .route(
-            &static_routes::v1::network::nym_contracts_detailed(),
+            "/nym-contracts-detailed",
             axum::routing::get(nym_contracts_detailed),
         )
 }
 
+#[utoipa::path(
+    tag = "network",
+    get,
+    path = "/v1/network/details",
+    responses(
+        (status = 200, body = NetworkDetails)
+    )
+)]
 async fn network_details(
     extract::State(state): extract::State<AxumAppState>,
 ) -> axum::Json<NetworkDetails> {
     state.network_details().to_owned().into()
 }
 
+// it's used for schema generation so dead_code is fine
+#[allow(dead_code)]
+#[derive(ToSchema)]
+#[schema(title = "ContractVersion")]
+pub(crate) struct ContractVersionSchemaResponse {
```

Comment:
why do we have to re-define that struct? couldn't we have used `ContractInformation` directly with the aliases?

Reply:
> couldn't we have used ContractInformation directly with the aliases

what do you mean by this?